### ISSUE TITLE
Set LIBVA_DRIVER_NAME

### ISSUE THREAD

**Issue by george-angel** [hertg/egpu-switcher#92]
Hello!

More of a question, feature request. Would it be possible to have egpu-switcher set `LIBVA_DRIVER_NAME`.

Currently the VA-API library tries to guess (incorrectly) which lib to use, from: https://wiki.archlinux.org/title/Hardware_video_acceleration#Configuration

```
$ grep -iE 'vdpau | dri driver' ~/.local/share/xorg/Xorg.0.log
[    32.508] (II) AMDGPU(0): [DRI2]   DRI driver: radeonsi
[    32.508] (II) AMDGPU(0): [DRI2]   VDPAU driver: radeonsi
[    32.681] (II) modeset(G0): [DRI2]   DRI driver: iris
[    32.681] (II) modeset(G0): [DRI2]   VDPAU driver: va_gl
```

```
$ vainfo
vaInitialize failed with error code -1 (unknown libva error),exit
```

```
$ LIBVA_DRIVER_NAME=radeonsi vainfo
vainfo: VA-API version: 1.16 (libva 2.15.0)
vainfo: Driver version: Mesa Gallium driver 22.2.1 for AMD Radeon RX 480 Graphics (polaris10, LLVM 14.0.6, DRM 3.48, 6.0.2-arch1-1)
vainfo: Supported profile and entrypoints
      VAProfileMPEG2Simple            : VAEntrypointVLD
      VAProfileMPEG2Main              : VAEntrypointVLD
      VAProfileVC1Simple              : VAEntrypointVLD
      VAProfileVC1Main                : VAEntrypointVLD
      VAProfileVC1Advanced            : VAEntrypointVLD
      VAProfileH264ConstrainedBaseline: VAEntrypointVLD
      VAProfileH264ConstrainedBaseline: VAEntrypointEncSlice
      VAProfileH264Main               : VAEntrypointVLD
      VAProfileH264Main               : VAEntrypointEncSlice
      VAProfileH264High               : VAEntrypointVLD
      VAProfileH264High               : VAEntrypointEncSlice
      VAProfileHEVCMain               : VAEntrypointVLD
      VAProfileHEVCMain               : VAEntrypointEncSlice
      VAProfileHEVCMain10             : VAEntrypointVLD
      VAProfileJPEGBaseline           : VAEntrypointVLD
      VAProfileNone                   : VAEntrypointVideoProc
```

Thank you.

**Comment by hertg** [comment]
Hey there @george-angel 

You could try setting the variable yourself by using a hook script. Simply create a shell script and configure egpu-switcher to run it when switching to the egpu.

*/etc/egpu-switcher/config.yaml*

```yaml
hooks:
  egpu: /path/to/script.sh
```

Note that the shell script needs to be owned by `root` and have a permission of `700`. This is just a precautionary requirement so that an unprivileged user can not manipulate the script since it will be run with root privileges.

Let me know if that works for you.

**Comment by george-angel** [comment]
Ok, sorry, I think I got tripped up in my own config.

tldr - I put this snippet in `~/.bash_profile` now to make things work:

```
if lspci -k | grep -q 'amdgpu'; then
  export LIBVA_DRIVER_NAME=radeonsi
elif lspci -k | grep -q 'i915'; then
  export LIBVA_DRIVER_NAME=i965
fi
```
